new/LIT7119Yethafaru.xml

```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7119Yethafaru" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ይትኃፈሩ፡ ወይኅሰሩ፡ ኵሎሙ፡ ጸላእትየ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Yǝtḫāfaru wa-y̮ǝḫsaru kʷǝllomu ṣolāʿǝtəya...</title>
+                <title xml:lang="en" corresp="#t1">All my enemies shall be ashamed and shall be disgraced...</title>
```

Comment:
The verbs are in the jussive, so perhaps "May all my enemies be ashamed and disgraced..."